docs/content/maintenance/write-performance.md

```diff
@@ -165,8 +165,10 @@ Run the following command to submit a compaction job for the table.
     /path/to/paimon-flink-**-{{< version >}}.jar \
     compact \
     --warehouse <warehouse-path> \
-    --database <database-name> \
-    --table <table-name>
+    --database <database-name> \ 
+    --table <table-name> \
+    [--partition <partition-name>] \
+    [--catalog-conf <paimon-catalog-conf> [--catalog-conf <paimon-catalog-conf> ...]] \
```

Comment:
Add description for catalog-conf, like:
'--catalog-conf' is the configuration for Paimon catalog. Each configuration should be specified in the format 'key=value'. See [here]({{< ref "maintenance/configurations" >}}) for a complete list of catalog configurations.